### PR TITLE
Map convergence

### PR DESCRIPTION
**Pull request by ethanjurman**
Uses a point map for the add point page! It's super cool :snowflake: 

Uses Redux to load and make state changes to the add point card.

### CODE REVIEW

File: www/js/components/add-point-card.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import HoursTable from './hours-table';
 // import redux components
 import { connect } from 'react-redux';
 
-import {addPoint} from '../actions/point-actions';
+import { addPoint } from '../actions/point-actions';
```
Comment (JRJurman) [inline www/js/components/add-point-card.js]: is this change required for this PR?

File: www/js/components/add-point-card.js [hunk 2]
```diff
@@ -57,11 +57,12 @@ export class AddPointCard extends Component {
   }
 
   onSubmit() {
+    const { mapReducer } = this.props;
```
Comment (JRJurman) [inline www/js/components/add-point-card.js]: mapReducer is probably a poor name choice...
is mapState a viable possibility?

**Comment by JRJurman** [comment]
Other than the 2 comments I made, it looks okay to me.
:tada: :cactus:

File: www/js/components/add-point-card.js (hunk 1 above)
Comment (ethanjurman) [inline www/js/components/add-point-card.js]: yas plz.
:ok_hand: ?
